drivers/ethernet/Kconfig.sam_gmac

```diff
@@ -15,7 +15,12 @@ if ETH_SAM_GMAC
 config ETH_SAM_GMAC_QUEUES
 	int "Number of active hardware TX and RX queues"
 	default 1
-	range 1 $(dt_node_int_prop_int,/soc/ethernet@40050088,num-queues)
+	range 1 1 if SOC_SERIES_SAM4E
+	range 1 3 if !SOC_SERIES_SAM4E && \
+		     !SOC_ATMEL_SAME70_REVB && \
+		     !SOC_ATMEL_SAMV71_REVB
+	range 1 6 if SOC_ATMEL_SAME70_REVB || \
+		     SOC_ATMEL_SAMV71_REVB
```

Comment:
TODO: Investigate how to deal with this (cc @galak)
https://github.com/zephyrproject-rtos/zephyr/pull/24057#discussion_r406572496

Reply:
@nandojve See #24256